### DISCUSSION TITLE
230713 최우선 페르소나 확정 회의 (10:30 ~ 16:30)

### DISCUSSION THREAD

**Discussion by somsom13** [woowacourse-teams/2023-shook#47] [회의]
## 회의 안건

최우선 페르소나를 확정한다.

### 문제점

- 데모데이 이후에 고려해봤었던 '일반인' 페르소나의 needs는 우리가 충족시켜줄 수 없다. 
	- need를 만족하기 위해서는 어떤 방법이든 곡 추천을 해야 한다. 그러나 우리가 추천을 해줄 수 있는 방법은 없다.
	- 우리가 직접적으로 추천을 해주지 않는다고 하더라도, 우리의 '간접적인' 추천은 (킬링 파트의 투표를 보고 그 음악이 좋다고 생각하든지 안 하든지 간에) 일반인 페르소나에게 pain point가 되지 않는다.
	- 일반인의 pain point를 해결하는 서비스는 시중에 너무나도 많다. (꼭 새로운 서비스가 아니어도 되지만, 우리만의 특별한 기능을 특별하게 꼽기 어렵다)
	- 일반인을 위한 기능을 제공해주기 어렵다. 듣기와 좋은 음악을 찾기 원하는 페르소나이다 보니, 일반인이 최우선 페르소나가 되는 경우 추천과 듣기에 집중된 기능을 개발해야 한다. 그러나 현실적으로 듣기가 매우 어렵고, 듣는 방법을 유튜브 링크로 한정하는 경우도 마찬가지로 일반인의 pain point를 채워줄 수 없다.
	- 일반인이 최우선 페르소나가 된다면 '킬링파트 듣기'가 최우선 개발 기능이 되어야 할 것 같다.
- 팬덤을 공략하는 방법
	- 우리의 주변에는 팬덤이 없기 때문에 실제로 팬덤이 어떤 니즈를 갖고 있는지 알 수 없다. 우리가 팬덤을 위해 어떤 것을 제공할 수 있는가? 팬덤이 원하는 것은 어떤 것인가?
	- 비교할 만한 표본이 없고, 다른 서비스에서 제공하는 것들 (1위를 하면 광고를 해준다던지) 을 제공하기는 어렵다. 
	- 이용자 수를 늘리기 위해 팬덤 측에 마케팅을 시도하는 것 자체는 좋지만, 팬덤이 페르소나이자 메인 이용자가 되어서는 안 된다고 생각한다. 우리는 팬덤의 니즈를 해결하기 어렵다. (아직 니즈가 뭔지도 잘 모르기 때문에...)

### 그래서...

페르소나는 보편적일 필요는 없다. 그러나 우리 주변에서 볼 수 있거나, 어딘가에서 봤거나, 들어봤어야 한다.

우리는 무슨 가치를 사용자에게 전달하고 싶나?
우리의 애플리케이션을 통해 어떤 것을 얻어 가기를 원하는가?
왜 사용자가 우리의 서비스를 사용할 수 밖에 없을까?

서비스의 사용을 구걸하지 말고, 자연스럽게 사용할 수 있도록 해야 한다.

### '킬링파트를 공유하는 것'이 우리의 사용자에게 어떤 가치를 전달할 수 있을까?

=> 타인에게 인정받고자 하는 욕구, 내가 추천한 노래를 다른 사람들이 좋아했을 때 기쁨을 느낀다. 내가 추천한 킬링파트를 다른 사람들이 인정할 때 (투표가 많이 달렸을 때)
=> 내가 음잘알이다. 내가 이렇게 힙한 노래를 많이 안다.
=> 누구나 아는 명곡보다는 나만의 명곡을 공유하는 경우가 많다.
=> 내가 음잘알인 것을 알리고 싶다.

-> 내가 추천한 노래를 다른 사람들이 들어보고 좋아했는지를 알려준다면?
-> 킬링파트 공유자의 인정욕/과시욕을 채워줄 수 있는 기능이 필요할 것 같다. 

### '킬링파트를 공유받는 것'이 우리의 사용자에게 어떤 가치를 전달할 수 있을까?

=> 도입부부터 들었다면 별로였을 노래가 좋은 노래라는 걸 알 수도 있다.
=> 처음 접하는 노래인 경우 좋은 노래인지 아닌지 다른 사람들의 보편적인 좋은 부분을 통해 바로 판단할 수 있다.
=> 만약 이전에 들어본 노래인데 별로라고 생각했던 경우 굳이 검색해서 안 찾아볼 거 같다.
=> 이전에 들어본 노래가 인기곡에 있다면 들어볼 듯. (인터뷰에 기반한 내용)
-> 노래를 검색해서 킬링파트를 찾아보게 하는 건 별로 좋지 않은 것 같다. 킬링파트를 공유받는 사람의 경우 주로 이전에 들어보지 않은 새로운 노래의 킬링파트를 듣는 것을 선호하지 않을까?
-> 인기곡으로 들어와서 다른 노래로 이어지게 하는 플로우는 어려울 수도 있다. 이 사람의 취향을 알지 못하면 그냥 아무 노래나 띄워줄 수밖에 없다.

이 노래가 내 취향인지 아닌지를 빠르게 파악한다. => 추천이 필요가 없다.

**Comment by Cyma-s** [comment]
## 페르소나

### 사람

킬링파트를 등록하고 공유하는 것이 메인인 페르소나

- **내가 좋아하는 부분을 사람들도 좋아한다는 걸 인정받고 싶다.**
    - 남들이 잘 모르는 노래를 추천했을 때의 인정 (독보적)
        - split, ukko
    - **내가 좋아하는 걸 다른 사람들도 좋아한다고 느낄 때의 인정 (보편적)**
        - dobob, baron, conan, split, ako, vero
        - 사람들이 좋아할 만한 부분을 공유 / 내가 좋아하는 걸 역시 다른 사람들도 좋아하네 → 대중적인 귀

### 가치를 어떻게 제공할까? (구현 생각 없이)

- **몇 명이 같은 부분을 좋아하는지 수치를 제공한다. +++-++**
- 내가 기여한 킬링파트가 TOP3에 얼마나 많이 들어갔나 (my 금은동)
- 음잘알 배지
- 좋아하는 사람들 중에서 내가 몇 번째(퍼센트)로 등록했는지 (빨리 등록하면 기분 좋음) |
- 유저 랭킹 (음잘알 랭킹) +
    - 같은 부분을 좋아하는 사람들이 가장 많은 순

**킬링파트: TOP3**

**파트: 그냥 등록한 구간**

### **몇 명이 같은 부분을 좋아하는지 수치를 제공**

- 등록한 파트에 몇 명이 공감하고 있는지 -
- 등록한 파트를 공감하는 비율이 노래의 총 파트 투표 중에 몇 퍼센트인지 +
- **내가 등록한 파트가 노래의 전체 파트 중에 몇 순위인지 +++++-**

코난 = 다 있으면 좋겠다

우코 = 2 + 3

### **내가 등록한 파트가 노래의 전체 파트 중에 몇 순위인지**

- 파트 등록
    - 구간 10초
    - 시작점 입력
- 순위 정보를 언제 보여줄 것인가?
    - 파트 등록 후, 바로 순위를 보여주자

### 등록한 파트를 어떻게 공유할까?

- 파트 등록 후, 순위 모달 창이 뜨면 아래 공유 링크
    - 파트 구간 공유
- 공유 방식
    - 내용 자체 (노래 정보, 파트) 공유할 지
    - **링크로 공유할 지 ++++++++**
        - 링크를 눌렀을 때의 화면
- 무슨 링크를 공유할까?
    - 음악을 들을 수 있는 외부 링크 (파트를 들어볼 수 있는 링크 ex. 유튜브 영상)
        - 유튜브 링크에 시작점을 찍어준다.
- 공유는 어디에 있나
    - 등록 버튼 누르면 모달 → 공유하기 버튼

### 나중에 생각할 거

- 지금은 1:10 ~ 1:20, 1:11 ~ 1:21 은 다른 구간이다. → 섬세 터치 필요
    - 우코: level 3가 지나기 전에 1:10~1:20, 1:11~1:21 가 다른지 정해야 하지 않을까?
    - 아코: 1:10~1:20, 1:11~1:21가 같으면 어떤 값으로 투표할 것인지 모르지 않을까?
- 로그인이 없기 때문에 파트 중복 등록 가능
- 파트 공유 시, 순위도 공유할까?
- 서비스 안에서 음악을 들을 수 있어야 한다
    - 해당 노래의 상세 페이지 링크(파트들을 볼 수 있는 S-HOOK 페이지)
    - 파트의 상세 페이지 링크
    - 이상향: 내가 정한 파트만 들을 수 있는 링크를 공유한다.
